src/core/files.js

```diff
@@ -58,26 +61,10 @@ export class FileStore {
    *
    * @param   {Query}                 [query]                                   Query used to filter result.
    * @param   {Object}                [options]                                 Options
-   * @param   {Properties}            [options.properties]                      Custom properties to send with
-   *                                                                            the request.
-   * @param   {Number}                [options.timeout]                         Timeout for the request.
-   * @param   {Boolean}               [options.tls]                             Use Transport Layer Security
-   * @param   {Boolean}               [options.download]                        Download the files
-   * @return  {Promise}                                                         Promise
-   *
-   * @example
-   * var filesStore = new Kinvey.FilesStore();
-   * var query = new Kinvey.Query();
-   * query.equalTo('location', 'Boston');
-   * files.find(query, {
-   *   tls: true, // Use transport layer security
-   *   ttl: 60 * 60 * 24, // 1 day in seconds
-   *   download: true // download the files
-   * }).then(function(files) {
-   *   ...
-   * }).catch(function(err) {
-   *   ...
-   * });
+   * @param   {boolean}               [options.tls=true]                        Use transport layer security
+   * @param   {number}                [options.ttl]                             Specify a time to live for the _downloadURL
+   * @param   {boolean}               [options.download=false]                  Download the files
```

Comment:
Similar to the TLS comment about. Needs some detail about why files aren't downloaded by default, where to find them if they are downloaded, etc.

Reply:
Good point. I will make the change.